themes/jaeger-docs/layouts/index.redirects

```diff
@@ -0,0 +1,11 @@
+{{- $latest := .Site.Params.versions.latest }}
+{{- $docs := readDir "content/docs/1.5" }}
+/docs/latest     /docs/{{ $latest }}
+/docs/latest/*     /docs/{{ $latest }}/:splat
```

Comment:
do we need these?  I don't think we ever had `/latest/` in the URLs.

Reply:
It's up to you if you want to have `latest` URLs. In general, it's very useful to be able to give people a URL that works permanently. If you release a new feature in 1.2 that's documented at `/docs/1.2/new-feature` and aliased at `/docs/latest/new-feature`, then the latter URL will continue to work through later versions. If someone only sees or has access to the `/docs/1.2/new-feature` URL then they'll be seeing outdated information if the project version is, say, 3.5.